mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -619,14 +622,18 @@ def __init__(self,
                  optimizer: Union[Optimizer, OptimWrapper],
                  param_name: str,
                  T_max: Optional[int] = None,
-                 eta_min: float = 0.,
+                 eta_min: Optional[float] = None,
+                 eta_min_ratio: Optional[float] = None,
```

Comment:
Adding a new parameter in the middle position will cause backward compatibility.

Reply:
In addition, changing the default value of `eta_min` to None also breaks backward compatibility.